src/main/java/org/openstreetmap/atlas/geography/atlas/items/AtlasEntity.java

```diff
@@ -194,6 +199,66 @@ public String toDiffViewFriendlyString()
      */
     public abstract LocationIterableProperties toGeoJsonBuildingBlock();
 
+    public abstract JsonObject asGeoJsonFeature();
+
+    /**
+     * A method that creates properties for a GeoJSON Feature from the tags.
+     *
+     * @return A GeoJSON properties object that is to be put in a Feature.
+     */
+    protected JsonObject geoJsonProperties()
+    {
+        final JsonObject properties = new JsonObject();
+        getTags().forEach(properties::addProperty);
+        properties.addProperty(GeoJsonUtils.IDENTIFIER, getIdentifier());
+        properties.addProperty(GeoJsonUtils.OSM_IDENTIFIER, getOsmIdentifier());
+        properties.addProperty(GeoJsonUtils.ITEM_TYPE, String.valueOf(getType()));
+
+        final Optional<String> shardName = getAtlas().metaData().getShardName();
+        shardName.ifPresent(shard -> properties.addProperty("shard", shard));
+
+        final Set<Relation> relations = relations();
+        if (relations.size() > 0)
+        {
+            final JsonArray relationsArray = new JsonArray();
+            properties.add("relations", relationsArray);
+            for (final Relation relation : relations)
+            {
+                final JsonObject relationObject = relation.geoJsonProperties();
+                relationsArray.add(relationObject);
+                final RelationMemberList members = relation.members();
+                final JsonArray membersArray = new JsonArray();
+                relationObject.add("members", membersArray);
+                for (final RelationMember member : members)
```

Comment:
My thought is that if you select a given relation member, which will have an associated geometry, having this encoded would allow a map viewer to select all of the members of a relation in the map viewer. If you do not do this, you will not have that ability.

We are not actually putting in the entity of the members, just the id, role, and type, as you would see in standard OSM XML. 

So, you do get that trade off.

I have tested out this change, and I'm not seeing a difference in how fast the vector tiles render.

I do see the disadvantage that there's too much stuff in the popup of the properties in mbview, but a custom map viewer would address this. 

I can take this out, but there is that trade off. You will no longer get the data that is a relation in your application.

Reply:
Considering this is a WIP PR, I think it's fine to leave it in for now - just to see how performance will look at scale, as well as test user experience regarding map viewing. This can always be adjusted/made configurable later.

Reply:
So any time I select one member of a relation all the other members will get selected too? Or will the map viewer give me an option?